lib/msal-react-native-android-poc/SampleApp/App.js

```diff
@@ -93,8 +117,9 @@ class App extends Component {
                 <Text style={styles.textSubHeader}> MSGraph URL: </Text>
                 <TextInput
                 style={styles.textInput}
-                placeholder="Some URL"
-                underlineColorAndroid='#8a8886' />
+                defaultValue='https://graph.microsoft.com/v1.0/me'
```

Comment:
I think making this a constant string (maybe at the top of the file if you only use it in App.js, or in a constants directory somewhere else that you can import, may save you a few headaches down the line.

Reply:
url string coded in one place (in the constructor as URLValue) and now the defaultValue prop references it